setup.py

```diff
@@ -101,34 +100,35 @@
         "ipython>=8.5.0,<9",
         "lazyasd>=0.1.4",
         "packaging>=23.0,<24",
-        "pandas>=1.3.0,<2",
+        "pandas>=2",
         "pluggy>=1.3,<2",
         "pydantic>=2.5.2,<3",
         "pydantic-settings>=2.0.3,<3",
         "PyGithub>=1.59,<2",
         "pytest>=6.0,<8.0",
-        "python-dateutil>=2.8.2,<3",
+        "python-dateutil>=2.8.2",
         "PyYAML>=5.0,<7",
         "requests>=2.28.1,<3",
-        "rich>=12.5.1,<14",
+        "rich>=12.5.1",
         "SQLAlchemy>=1.4.35",
         "tqdm>=4.62.3,<5.0",
         "traitlets>=5.3.0",
         "urllib3>=2.0.0,<3",
         "watchdog>=3.0,<4",
         # ** Dependencies maintained by Ethereum Foundation **
         "eth-abi>=5.1.0,<6",
-        "eth-account>=0.10.0,<0.11",
+        "eth-account>=0.10.0",
+        "eth-tester @ git+https://github.com/wakamex/eth-tester.git",
         "eth-typing>=3.5.2,<4",
-        "eth-utils>=2.3.1,<3",
+        "eth-utils>=2.3.1",
         "py-geth>=4.4.0,<5",
-        "web3[tester]>=6.16.0,<7",
+        "web3[tester]>=6.16.0",
         # ** Dependencies maintained by ApeWorX **
-        "eip712>=0.2.3,<0.4",
+        "eip712 @ git+https://github.com/wakamex/eip712.git",
```

Comment:
- [ ] replace with released versions

Reply:
done by resetting dependencies (moved to another PR in https://github.com/ApeWorX/ape/pull/2003) in f04d70289737d69453a356398544c917cb547be5